frontend/src/pages/LoginPage.tsx

```diff
@@ -57,15 +61,26 @@ function LoginPage() {
             {...register('password', loginDataValidation.password)}
           />
           <FormError message={errors.password?.message} />
-          <input
-            className='btn btn-info mt-5 w-1/2 mx-auto'
-            type='submit'
-            value='Login'
-          />
+          <button className='btn btn-info mt-5 w-1/2 mx-auto' type='submit'>
+            {props.loading ? (
+              <div className='flex items-center justify-center'>
+                <div className='w-4 h-4 border-b-2 border-white-900 rounded-full animate-spin mr-5'></div>
+                Login
+              </div>
```

Comment:
Please use `<span/>` instead of `<div/>` according to w3c it is not valid.
check it in [w3c validator](https://validator.w3.org/nu/#textarea)